packages/api/tests/api/gitHub.test.ts

```diff
@@ -0,0 +1,28 @@
+import "jest";
+import { gitHub  } from "../../src/api/auth/gitHub";
+import { testHandler } from "../testUtils/testHandler";
+import { env } from '../../src/env';
+
+const GitHubClientId = env.GitHubId;
+
+const url = 'https://github.com/login/oauth/authorize?client_id=${GitHubClientId}&redirect_uri=http://localhost:3000/api/auth/github/callback';
```

Comment:
You don't need to get it from the environment, you can just pick a value since it's all mocked

```suggestion
const url = 'https://github.com/login/oauth/authorize?client_id=mock-client-id&redirect_uri=http://localhost:3000/api/auth/github/callback';
```